lib/sarif/bandit_sarif.rb

```diff
@@ -0,0 +1,60 @@
+module Sarif
+  class BanditSarif < BaseSarif
+    include Salus::SalusBugsnag
+
+    BANDIT_URI = 'https://github.com/PyCQA/bandit'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @uri = BANDIT_URI
+      @logs = parse_scan_report!
+    end
+
+    def parse_scan_report!
+      parsed_result = JSON.parse(@scan_report.log(''))
+
+      parsed_result['results'].concat(parsed_result['errors'])
+    rescue JSON::ParserError => e
+      bugsnag_notify(e.message)
+      []
+    end
+
+    def parse_error(error)
+      id = error['filename'] + ' ' + error['reason']
+      return nil if @issues.include?(id)
+
+      @issues.add(id)
+      {
+        id: 'SAL0002',
```

Comment:
Let's put this error code in a descriptive constant and use that whever we have SAL0002 used